src/mmw/apps/modeling/mapshed.py

```diff
@@ -219,10 +222,10 @@ def point_source_discharge(geom, area):
         cursor.execute(sql, [geom.wkt])
         mg_d, kgn_month, kgp_month = cursor.fetchone()
 
-        n_load = np.array([kgn_month] * 12)
-        p_load = np.array([kgp_month] * 12)
+        n_load = np.array([kgn_month] * 12) if kgn_month else np.zeros(12)
+        p_load = np.array([kgp_month] * 12) if kgp_month else np.zeros(12)
         discharge = np.array([float(mg_d * days * LITERS_PER_MGAL) / area
-                              for days in MONTHDAYS])
+                              for days in MONTHDAYS]) if mg_d else np.zeros(12)
```

Comment:
Well, I want to use three different arrays, all initialized to 0, not pass around the reference to one array, even though they are all identical.